apps/pkeyutl.c

```diff
@@ -310,7 +310,11 @@ int pkeyutl_main(int argc, char **argv)
         goto opthelp;
     } else if (peerkey != NULL && pkey_op != EVP_PKEY_OP_DERIVE) {
         BIO_printf(bio_err,
-                   "%s: no peer key given (-peerkey parameter).\n", prog);
+                   "%s: -peerkey option needlessly given (when not performing -derive).\n", prog);
```

Comment:
can we simplify wording here? how about:
```
"%s: -peerkey option not allowed without -derive.\n"
```

Reply:
I like your suggested improvement - done.